### PR TITLE
Few minor WP doc updates

### PR DESCRIPTION
**Pull request by jspellman814**
## Summary

Couple of edits due to Slack discussion: https://pantheon.slack.com/archives/CU174QD1B/p1680705318054619

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [x] Approval from a member of CMSP

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by whitneymeredith** [comment]
@jspellman814 Can this be converted from draft and merged?

**Comment by jspellman814** [comment]
Sure I'll convert it from a draft, but I'd like @pantheon-systems/cms-platform to review before we move forward. Thanks!

File: source/content/guides/multisite/01-introduction.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Pantheon supports the most common use case for WordPress Multisites: a common co
 - Blogs for faculty at a university
 - Franchise sites under a parent organization site
 - Sections within a media publication
-- [Bedrock based](https://carlalexander.ca/using-bedrock-with-pantheon/) or Composer based WP Multi-site is a possibility and can be discussed during onboarding
+- [Bedrock based](/guides/wordpress-composer/pre-ga/wordpress-composer-managed) or Composer based WP Multi-site is a possibility and can be discussed during onboarding
```
Comment (CdrMarks) [inline source/content/guides/multisite/01-introduction.md:31]:
```suggestion
- [Bedrock based](/guides/wordpress-composer/pre-ga/wordpress-composer-managed) or Composer based WP Multisite is a possibility and can be discussed during onboarding
```

**Review by CdrMarks — APPROVED** [review]

**Comment by jspellman814** [comment]
All set @whitneymeredith

**Review by whitneymeredith — APPROVED** [review]
LGTM

**Pull request merged**